flow/designs/gf180/uart-blocks/uart_rx/BLOCKS_grid_strategy.tcl

```diff
@@ -0,0 +1,24 @@
+####################################
```

Comment:
Why do we have both this and flow/designs/gf180/uart-blocks/BLOCKS_grid_strategy.tcl  ?

Reply:
For hardening max routing layer should be Metal4.
In the top design Metal5 used as max routing layer.
So PDN adjusted accordingly with different strategy